src/constants/responses.ts

```diff
@@ -23,3 +23,5 @@ export const RETRY_COMMAND =
   "Oops, we didn't catch that! Please use the command again.";
 
 export const ROLE_ADDED = "Role added successfully";
+
+export const NAME_CHANGED = "User nickname change successful";
```

Comment:
The statement `User nickname changed successfully` sounds better as a response and we have also followed similar responses previously. That is better I feel, isn't it?

Reply:
I am not sure, both convey the same message.

Reply:
Yes, but one looks like an action response and the other seems like a status update. Just a small thing